### PR TITLE
Simple Search Help

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
image

### CODE REVIEW

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/display.tsx [hunk 1]
```diff
@@ -65,9 +65,19 @@ export default class DirectoryDisplay extends React.Component<IProps> {
...
+            basic
+            hoverable
+            trigger={<div>
+              <DebouncedSearch
+                placeholder={t('common.search')}
+                onSubmit={this.search} />
+            </div>}
+            position='bottom center'>
+
+            <div dangerouslySetInnerHTML={{ __html: t('directory.search-help') }} />
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/display.tsx]: couldn't you just use "content property" of Popup?

I know you did it to add the <br> into it (but IMHO: HTML shouldn't be part of the translation file)

**Comment by NullVoxPopuli** [comment]
@giancorzo  the HTML _must_ be part of the translation file, otherwise I can't get a bulleted list :-\

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/display.tsx (hunk 1 above)
Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/display.tsx]: see top-level comment

Comment (giancorzo) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/display.tsx]: can do simple <li> html? if not go ahead and merge it

Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/display.tsx]: That'd be a lot more markup

**Pull request merged**